modules/apps/application-list/application-list-user-personal-site-permissions/src/main/java/com/liferay/application/list/user/personal/site/permissions/internal/model/listener/CompanyModelListener.java

```diff
@@ -57,6 +76,21 @@ public void onAfterCreate(Company company) {
 			});
 	}
 
+	@Activate
+	protected void activate(BundleContext bundleContext) {
+		_userPersonalSitePermissions = new UserPersonalSitePermissions(
```

Comment:
please open the ServiceTracker here and pass it as a constructor parameter instead.